ietf-eth-te-topology.yang

```diff
@@ -1,67 +1,96 @@
 module ietf-eth-te-topology {
-
+  yang-version 1.1;
   namespace "urn:ietf:params:xml:ns:yang:ietf-eth-te-topology";
-
-  prefix "ethtetopo";
+  prefix "eth-tet";
```

Comment:
would it be "etht" if aligned with the conventions for TE-based models?
We currently have: otnt, flexgt, tet, and oit...

Reply:
Prefix changed to 'etht'

See: https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-eth-client-te-topo-yang/pull/6/commits/2c8cb26b2e74d0958587d6db2c832526973c2b46